tempe/create_bwa_genome_index.sh

```diff
@@ -80,18 +80,18 @@ echo >> README
 echo "BWA Index creation details:" >> README
 echo >> README
 
+# Determine the expected FASTA sequence filename based on the download link
+FASTA=$REFERENCE_DNA_GENOME_NAME
+
+# Create a symbolic link to the reference genome
+ln -s ../../genome_reference/${FASTA} ${FASTA}
+
 # Copy in the .alt file from bwa.kit
 echo "Copy in the .alt file from bwa.kit" >> README
-cp ../../genome_reference/downloads/bwa.kit/resource-GRCh38/hs38DH.fa.alt GRCh38tgen_decoy_alts_hla.fa.alt
```

Comment:
Is this how we did it before, so the user needs to have independently pulled bwakit?

Reply:
We pull bwa.kit during the create_genome_reference.sh script that runs before this, though maybe we want to change this to be a path defined in the ini? e.g. PATH_TO_BWAKIT="/some/path" and we navigate the known structure and or we use/ask for the full path to the alt.

Future implementation might be to create the alt ourselves, though the curated file in bwa.kit is likely more "correct" than an automated approach.